### DISCUSSION TITLE
glTF Working Group Meeting 2021-11-04

### DISCUSSION THREAD

**Discussion by robertlong** [omigroup/gltf-extensions#31] [General]
# Agenda
- Ratification Process
  - Vote on the 4 Stage Proposal
    - https://github.com/omigroup/gltf-extensions/pull/23 
- Update on Implementation Status
  - Spec
    - Need to add markdown explainer
    - Publish blog post advertising the OMI_audio_emitter ecosystem
  - omi-gltf-samples-repo
  - three-omi
    - Stable and up to date
    - Transitioned to typescript
  - Third Room
    - Support is finished and merged: https://github.com/thirdroom/thirdroom/pull/17
  - Unity
    - https://github.com/omigroup/gltf-extensions/discussions/11
    - Got a responses back about glTFast:
      - https://github.com/omigroup/gltf-extensions/discussions/30#discussioncomment-1557443
      - https://github.com/atteneder/glTFast/issues/226#issuecomment-956421197
    - Discussion in Discord in the last week (need a summary)
  - Godot
    - https://github.com/godotengine/godot-proposals/issues/3273
    - https://github.com/godotengine/godot/pull/53738
    - https://github.com/godotengine/godot/pull/52407
    - https://github.com/godotengine/godot/pull/52541
  - Blender
    - https://github.com/omigroup/gltf-extensions/discussions/26 
    - https://github.com/omigroup/omi-blender-gltf/pull/1
  - VRChat
    - Blocked on OMI Unity glTF Editor Import
  - Altspace
    - Blocked on OMI Unity glTF Editor Import
  - Neos
    - Looking into MP3 player support and accessing proper data on scene nodes
  - Hubs
    - PR submitted to Hubs repo https://github.com/mozilla/hubs/pull/4741  
  - glTF Validator
    - Up to date with the latest spec 
  - glTF Transform Tool
    - Up to date with the latest spec 
  - Stereo Kit
    - Waiting on an update from Nova
- Next extensions
  - When is it appropriate to start working on these?
  - What resources would make this process easier?
    - File issues for various resources needed
  - What proposals would people like to see most?

**Comment by robertlong** [comment]
# Notes
- Unity
  - Made our own OMI fork of glTFast https://github.com/omigroup/glTFast
  - Need to chime in on https://github.com/atteneder/glTFast/issues/226
- Godot
  - 3.4 should be out tomorrow
  - Need to see what glTF import/export features are a part of that release
- Blender
  - Import and Export plumbing mostly there
  - Not exporting bufferView or moving audio files to export path, everything else basically working
  - Needs testing afterwards
- 4 Stage Proposal Process
 - Passed and merged into the repo
- [OMI_texture_materialx](https://github.com/fire/glTF/tree/materialx/extensions/2.0/Vendor/EXT_texture_mtlx)
  - https://github.com/fire/MaterialX/pull/1
- Group is ready to start working on additional extensions
  - https://github.com/omigroup/gltf-extensions/discussions

**Reply by fire** [reply]
I have updated the url to https://github.com/fire/glTF/tree/materialx/extensions/2.0/Vendor/OMI_texture_mtlx
